conformance/src/main/scala/za/co/absa/enceladus/conformance/DynamicConformanceJob.scala

```diff
@@ -247,6 +244,25 @@ object DynamicConformanceJob {
     }
   }
 
+  private def handleEmptyOutputAfterConformance()(implicit spark: SparkSession): Unit = {
```

Comment:
Tiny: Wouldn't be a `checkEmptyOutputAfterConformance` be a better suiting name?

Reply:
No, because we know it is empty. That is why we are calling this method.